db/seeds.rb

```diff
@@ -41,7 +41,8 @@
   park_id: 2
 )
 Review.create!(
-  rating: 2,
+  rating: 0,
+  body: 'MY EYES ARE BLEEDING, FIRE EVERYWHERE, AAAAAAAAAAAH',
   park_id: 1,
   user_id: 1
 )
```

Comment:
um...while this is hilarious, hiring partners will see it. I miiiiggghhhttt update it. It can be quirky, but ideally not violent haha